### PR TITLE
Added selection_sort.cpp to C++ folder

### PR DESCRIPTION
**Pull request by Saujanya0910**
## Have you read the Contribution Guidelines?

Yes

## Description

Added selection_sort.cpp to C++ folder 

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue 
<br/>
#54

### CODE REVIEW

File: C-Plus-Plus/selection_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    cout << "Array before Sorting: ";
+    display(arr, n);
+
+    // sorting the array 
+    selectionSort(arr, n);
+
+    cout << "Array after Sorting: ";
+    display(arr, n);
+}
```
Comment (sukritishah15) [inline C-Plus-Plus/selection_sort.cpp:63]: Add below this, 
within single multiline comment
1. Sample I/P
2. Sample O/P
3. Time Complexity
4. Space Complexity


Also, add your file link in README.md file inside C++ folder.
Also, work from a different branch always. Not master branch.

Comment (Saujanya0910) [reply C-Plus-Plus/selection_sort.cpp:63]: Sample I/P: 
Enter the number of elements: 6                                                                                                               
Enter elements:                                                                                                                               
9 11 5 7 4 10

Sample O/P:
Array before Sorting: 9 11 5 7 4 10                                                                                                           
Array after Sorting: 4 5 7 9 10 11

Time Complexity: O(n^2)

Space Complexity: O(1)

Comment (Saujanya0910) [reply C-Plus-Plus/selection_sort.cpp:63]: So I'll have to make another PR to change the README right?

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -10,3 +10,4 @@ Format:
 
 -[Sieve of Eratosthenes](sieve_of_eratosthenes.cpp)
 
+-[Selection Sort](selection_sort.cpp)
```
Comment (Saujanya0910) [inline C-Plus-Plus/README.md:13]: Added selection_sort.cpp link to README.md

Comment (Saujanya0910) [inline C-Plus-Plus/README.md:13]: Added selection_sort.cpp link to README.md

File: C-Plus-Plus/selection_sort.cpp (hunk 1 above)
Comment (Saujanya0910) [reply C-Plus-Plus/selection_sort.cpp:63]: Sample I/P:
Enter the number of elements: 6
Enter elements:
9 11 5 7 4 10

Sample O/P:
Array before Sorting: 9 11 5 7 4 10
Array after Sorting: 4 5 7 9 10 11

Time Complexity: O(n^2)

Space Complexity: O(1)

Comment (sukritishah15) [reply C-Plus-Plus/selection_sort.cpp:63]: I meant in the code, not as a reply, here. xD
Nevermind, I'll merge this PR and add it for you.

Comment (Saujanya0910) [reply C-Plus-Plus/selection_sort.cpp:63]: Sorry, I misinterpreted "Add below this, within single multiline comment". Thank you!

**Pull request merged**